rke/k8s_service_options.go

```diff
@@ -140,6 +140,22 @@ func loadK8sVersionServiceOptions() map[string]v3.KubernetesServicesOptions {
 			Kubeproxy:      getKubeProxyOptions(),
 			Scheduler:      getSchedulerOptions(),
 		},
+		"v1.15.12-rancher1-1": {
+			Etcd:           getETCDOptions(),
+			KubeAPI:        getKubeAPIOptions115(),
+			Kubelet:        getKubeletOptions115(),
+			KubeController: getKubeControllerOptions(),
+			Kubeproxy:      getKubeProxyOptions(),
+			Scheduler:      getSchedulerOptions(),
+		},
+		"v1.15.12-rancher2-2": {
```

Comment:
Why wasn't this added as part of @ryansann adding of 1.15.12? Why is it added now?

Reply:
I'm guessing we missed it, but we do need it because the fallback default option for v1.15 doesn't have etcd options